### PR TITLE
Create StorageObjectDataWorkflows-simple

### PR DESCRIPTION
**Pull request by minyu19**
Add sample data file for Eventarc

### CODE REVIEW

File: testdata/google/events/cloud/storage/v1/StorageObjectDataWorkflows-simple [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (grant) [inline testdata/google/events/cloud/storage/v1/StorageObjectDataWorkflows-simple:1]: - File must end with `.json`
- Just include the contents of `data`

**Review by grant — CHANGES_REQUESTED** [review]

**Review by minyu19 — COMMENTED** [review]
Thank you, Grant. I have updated the file as suggested.

**Review by grant — APPROVED** [review]
Ensure there is 2 space indentation. Approved

**Review by ace-n — APPROVED** [review]

**Comment by grant** [comment]
It looks like the CI: `Generate / build (pull_request)` is failing.

Looking into the CI logs, it shows the filename is invalid.

The filename for test data should be something like:

`testdata/google/events/cloud/storage/v1/StorageObjectData-workflows.json`

---

See https://github.com/googleapis/google-cloudevents/blob/main/testdata/README.md#test-data

**Comment by grant** [comment]
Also would be good to add this to the registry:

https://github.com/googleapis/google-cloudevents/blob/main/jsonschema/google/events/cloud/storage/v1/StorageObjectData.json

See failing `Generate / build (pull_request)` logs:

```
b/jsonschema/google/events/cloud/storage/v1/StorageObjectData.json
@@ -3,7 +3,8 @@
   "name": "StorageObjectData",
   "examples": [
     "https://googleapis.github.io/google-cloudevents/testdata/google/events/cloud/storage/v1/StorageObjectData-complex.json",
-    "https://googleapis.github.io/google-cloudevents/testdata/google/events/cloud/storage/v1/StorageObjectData-simple.json"
+    "https://googleapis.github.io/google-cloudevents/testdata/google/events/cloud/storage/v1/StorageObjectData-simple.json",
+    "https://googleapis.github.io/google-cloudevents/testdata/google/events/cloud/storage/v1/StorageObjectDataWorkflows-simple.json"
   ],
```

**Comment by minyu19** [comment]
This change had been initially suggested to refer to in the documentation. Later on, it was decided to add the appropriate examples to a different folder for documentation: https://github.com/googleapis/google-cloudevents/tree/main/examples. Since the samples have been added to the examples folder, I'm closing this PR.

**Pull request closed without merging**